sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_web_pub_sub_service_client.py

```diff
@@ -0,0 +1,94 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from copy import deepcopy
+from typing import TYPE_CHECKING
+
+from azure.core import PipelineClient
+from msrest import Deserializer, Serializer
+
+from ._configuration import WebPubSubServiceClientConfiguration
+from .operations import HealthApiOperations, WebPubSubOperations
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any, Dict, Optional
+
+    from azure.core.credentials import TokenCredential
+    from azure.core.rest import HttpRequest, HttpResponse
+
+class WebPubSubServiceClient(object):
+    """WebPubSubServiceClient.
+
+    :ivar health_api: HealthApiOperations operations
+    :vartype health_api: azure.messaging.webpubsubservice.operations.HealthApiOperations
+    :ivar web_pub_sub: WebPubSubOperations operations
+    :vartype web_pub_sub: azure.messaging.webpubsubservice.operations.WebPubSubOperations
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: ~azure.core.credentials.TokenCredential
+    :keyword endpoint: Service URL. Default value is ''.
+    :paramtype endpoint: str
+    """
+
+    def __init__(
+        self,
+        credential,  # type: "TokenCredential"
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        endpoint = kwargs.pop('endpoint', "")  # type: str
```

Comment:
@johanste autorest is generating this way because there's no host / endpoint param defined in the [webpubsub swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/webpubsub/data-plane/WebPubSub/preview/2021-08-01-preview/webpubsub.json#L6). I feel like there should be an endpoint param. Let me know and I'll make a quick PR to fix this swagger. thanks!

Reply:
> overall @msyyc Yuchao Yan FTE since a lot of this API is handwritten, generating autorest deleted a lot of the handwritten code. We don't want to delete any of the handwritten code, can you go back and add it back? thanks!

Got it. I add necessary code back, including `from_connection_string`

Reply:
Yeah, there should be an endpoint param for webpubsub.

Reply:
I also think `endpoint` is necessary. Could you help to make a quick PR? @iscai-msft

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/16001